tests/signature_tests/frequency_tests/test_traffic_freq_signatures.py

```diff
@@ -0,0 +1,140 @@
+import unittest
+import unittest.mock as um
+from scapy.layers.l2 import Ether
+from scapy.layers.inet import IP, UDP, TCP
+from src.anamoly_detection.frequency_signatures.traffic.traffic_layer_frequency_signature \
+    import TrafficLayerFrequencySignature
+from src.anamoly_detection.equation_parser import parse_equation
+
+root_path_test_data = "../../test_data/"
```

Comment:
Use Path instead so tests can be ran on windows

Reply:
Var itself isn't needed in this test, removing